packages/location-field/src/types.ts

```diff
@@ -214,6 +214,10 @@ export interface LocationFieldProps {
    * Results are sorted by distance, but favored layers will always appear first.
    */
   preferredLayers?: string[];
+  /**
+   * If true, make "Other" category (addresses, POIs, etc.) first in the results container.
+   */
+  renderOtherFirst?: boolean;
```

Comment:
Can we make this an array instead? Where we can manually set an order of categories to prioritize (and those will be displayed first). This might require a bigger refactor but I think it's worth it. Let me know if you'd like to discuss in more detail offline

Reply:
yeah!

Reply:
Good one

Reply:
Update in [03c02ac](https://github.com/opentripplanner/otp-ui/pull/771/commits/03c02acc508c472d78d87ec11e538f786b221fdc)!!